launcher/MTPixmapCache.h

```diff
@@ -100,22 +101,32 @@ class PixmapCache final : public QObject {
      */
     bool _markCacheMissByEviciton()
     {
+        static constexpr int maxInt = std::numeric_limits<int>::max();
+        static constexpr uint step = 10240;
+        static constexpr int oneSecond = 1000;
+
         auto now = QTime::currentTime();
         if (!m_last_cache_miss_by_eviciton.isNull()) {
             auto diff = m_last_cache_miss_by_eviciton.msecsTo(now);
-            if (diff < 1000) {  // less than a second ago
+            if (diff < oneSecond) {  // less than a second ago
                 ++m_consecutive_fast_evicitons;
             } else {
                 m_consecutive_fast_evicitons = 0;
             }
         }
         m_last_cache_miss_by_eviciton = now;
         if (m_consecutive_fast_evicitons >= m_consecutive_fast_evicitons_threshold) {
-            // double the cache size
-            auto newSize = _cacheLimit() * 2;
-            qDebug() << m_consecutive_fast_evicitons << "pixmap cache misses by eviction happened too fast, doubling cache size to"
-                     << newSize;
-            _setCacheLimit(newSize);
+            // increase the cache size
+            uint newSize = _cacheLimit() + step;
+            if (newSize >= maxInt) {  // increase it until you overflow :D
+                newSize = maxInt;
+                qDebug() << m_consecutive_fast_evicitons
+                         << tr("pixmap cache misses by eviction happened too fast, doing nothing as the cache size reached it's limit");
+            } else {
+                qDebug() << m_consecutive_fast_evicitons << tr("pixmap cache misses by eviction happened too fast, doubling cache size to")
```

Comment:
These seem to be the wrong way around? Also this message still says `doubling`

Reply:
Fixed the message.
But can you please be more explicit about what is wrong(I usually do not get the small details)?